tests/TestCase/ORM/Behavior/TreeBehaviorTest.php

```diff
@@ -427,7 +427,7 @@ public function testMoveLeaf(): void
         $table = $this->getTableLocator()->get('MenuLinkTrees');
         $table->addBehavior('Tree', ['scope' => ['menu' => 'main-menu']]);
         $node = $table->moveUp($table->get(5), 1);
-        $this->assertEquals(['lft' => 6, 'rght' => 7], $node->extract(['lft', 'rght']));
+        $this->assertEquals(false, $node);
```

Comment:
Should these be updated to `assertSame()` or `asertFalse()` so we don't have loose comparisons to null passing?

Reply:
I will do that now